cli/src/lib.rs

```diff
@@ -92,8 +92,10 @@ impl Cli {
 
     pub fn run_inner(self) -> Result<()> {
         if let Some(filter_path) = self.filter.as_ref() {
-            use std::fs::File;
-            use std::io::{BufRead, BufReader};
+            use std::{
+                fs::File,
+                io::{BufRead, BufReader},
+            };
```

Comment:
This was a product of running `cargo fmt`. Not sure if we want to keep (I wouldn't write it like this)